app/components/administrative-unit-status.hbs

```diff
@@ -0,0 +1,3 @@
+<AuPill @skin={{this.getStatusSkin}}>
+  {{@status}}
```

Comment:
I think it might be better to pass in the full "organization-status-code" record and let the component handle the rendering instead of having to pass the specific label in.